Framework/DataHandling/src/LoadSpec.cpp

```diff
@@ -86,12 +85,13 @@ void LoadSpec::exec() {
 
     // line with data, need to be parsed by white spaces
     if (!str.empty() && str[0] != '#') {
-      typedef boost::tokenizer<boost::char_separator<char>> tokenizer;
-      boost::char_separator<char> sep(" ");
-      tokenizer tok(str, sep);
-      for (tokenizer::iterator beg = tok.begin(); beg != tok.end(); ++beg) {
+      typedef Mantid::Kernel::StringTokenizer tokenizer;
+      const std::string sep = " ";
+      tokenizer tok(str, sep,
+                    Mantid::Kernel::StringTokenizer::TOK_IGNORE_EMPTY);
+      for (const auto &beg : tok) {
         std::stringstream ss;
-        ss << *beg;
+        ss << beg;
         double d;
         ss >> d;
         input.push_back(d);
```

Comment:
Would using [`std::stod`](http://www.cplusplus.com/reference/string/stod/) be more clear?